selfdrive/car/chrysler/values.py

```diff
@@ -177,6 +187,47 @@ class ChryslerCarInfo(CarInfo):
       b'68500483AB',
     ],
   },
+
+  CAR.RAM_HD: {
+    (Ecu.combinationMeter, 0x742, None): [
+      b'68361606AH',
+      b'68492693AD',
+      b'68525485AB', #TODO: CHECK in newest updat on dongle 5c05760b592aee2c
+    ],
+    (Ecu.srs, 0x744, None): [
+      b'68399794AC',
+      b'68428503AA',
+      b'68428505AA',
+    ],
+    (Ecu.esp, 0x747, None): [
+      b'68334977AH',
+      b'68504022AB',
+      b'68530686AB',
+    ],
+    (Ecu.fwdCamera, 0x753, None): [
+      b'04672895AB',
+      b'56029827AG',
+      b'68484694AE',
+    ],
+    (Ecu.eps, 0x761, None): [
+      b'68421036AC',
+      b'68507906AB',
+    ],
+    (Ecu.engine, 0x7e0, None): [
+      b'52421132AF',
+      b'M2370131MB',
+      b'M2421132MB',
+    ],
+    #(Ecu.transmission, 0x7e1, None): [
+      #b'68475153AD',
+      #b'68504640AC',
+      #b'68538041AB',
+    #],
```

Comment:
why is this commented out?

Reply:
I do not feel the transmission is important for fingerprinting. Also on a truck with a 68rfe, the tcm logic is housed in the ecm. It’s considered a pcm. There will be no transmission fingerprint if I remember correctly.